app/src/main/java/ar/com/wolox/android/cookbook/scanqr/scanerror/ScanErrorFragment.kt

```diff
@@ -8,14 +8,15 @@ import javax.inject.Inject
 
 class ScanErrorFragment @Inject constructor() : WolmoFragment<ScanErrorPresenter>() {
     override fun init() {
-        setOnClickListeners()
+        setListeners()
     }
 
     override fun layout(): Int = R.layout.fragment_scan_error
 
-    private fun setOnClickListeners() {
+    override fun setListeners() {
+        super.setListeners()
         bGoToScanMenu.setOnClickListener {
```

Comment:
I assume that bGoToScanMenu is a button. I know you didn't changed this, but it's a good idea to keep in mind that that's not a good practice to name components :p you can change that name to vScanMenuButton or something like that